### DISCUSSION TITLE
livereload seems to require make.bat

### DISCUSSION THREAD

**Discussion by homespuneffects** [talkpython/static-sites-with-sphinx-and-markdown-course#4] [General]
This is probably common knowledge but for the noobs like me, getting this to work on Windows in VS Code requires 'make.bat html' for it to work.

```
from livereload import Server, shell

if __name__ == '__main__':
    server = Server()
    server.watch('*.rst', shell('make.bat html'), delay=1)
    server.watch('*.md', shell('make.bat html'), delay=1)
    server.watch('*.py', shell('make.bat html'), delay=1)
    server.watch('*._static/*', shell('make.bat html'), delay=1)
    server.watch('*._templates/*', shell('make.bat html'), delay=1)
    server.serve(root='_build/html')
```

**Comment by pauleveritt** [comment]
Thanks for making a note of that. After making the original tutorial I learned about `sphinx-autobuild` which I should encourage people to use instead of the script above.

**Reply by homespuneffects** [reply]
Ah, excellent. I'll go check it out.

**Reply by pauleveritt** [reply]
Here is [the link](https://github.com/executablebooks/sphinx-autobuild)

When you install it, you get a command-line script available in your virtual environment. Running it then makes a server listening on a local URL.

**Reply by homespuneffects** [reply]
Thank you. :)

**Reply by aj-white** [reply]
For those wondering the cli command to get sphinx-autobuild to work is
`sphinx-autobuild path/to/project/directory _build/html`
